bindings/python/iota_sdk/types/address.py

```diff
@@ -105,12 +105,12 @@ class RestrictedAddress:
     type: int = field(default_factory=lambda: int(
         AddressType.RESTRICTED), init=False)
 
-    def with_allowed_capabilities(self, allowed_capabilities: bytes):
+    def with_allowed_capabilities(self, capabilities: bytes):
         """Sets the allowed capabilities from a byte array.
         Attributes:
-            allowed_capabilities: The allowed capabilities bitflags.
+            capabilities: The allowed capabilities bitflags.
         """
-        self.allowed_capabilities = '0x00' + allowed_capabilities.hex()
+        self.allowed_capabilities = hex(len(capabilities)) + capabilities.hex()
```

Comment:
I'm not fully sure the length is required?

Reply:
why not?

Reply:
We just don't put lengths in hex, I've confirmed with Phillip that it should be without length

Reply:
There is no real need for it, it's parseable without

Reply:
ohhhh duh